### ISSUE TITLE
Source code

### ISSUE THREAD

**Issue by Julienraptor01** [TheMysterys/Server-Pack-Unlocker#2]
The original repo gave the source code, could you give it too ?

**Comment by UltraBlackLinux** [comment]
where source ._.

How would I know that this isn't malware?

**Comment by Julienraptor01** [comment]
> This repo was never made to provide source code but more as a way for people to contact me for support/request features.
> 
> As for the question on whether the mod is malware, the mod is easily decompilable to view the source code if one truly wishes to do so.

It's not the on the point of seeing if it's malware, it's mostly for QoL if someone wanted to contribute to the mod, since currently the only way to contribute is to suggest things

**Comment by LiamDun** [comment]
you fr using a github repo for a minecraft mod that had to be programmed, but for "people to contact you".

not only is that extremely suspicious because it means the mod can literally have malware in it and nobody would know,
but it's also not your mod? You straight up took [Jackson's Mod](https://github.com/jaacksondev/server-pack-unlocker), and used the same code, icon, and name without any mention of his mod. Do you realize forking exists?

**Comment by TheMysterys** [comment]
Originally there was no way to contact me due to the fact that Modrinth does not have a comment section. I'm away from home for the next 3 months (no access to personal computer which contains the source code). 

The reason this repo doesn't fork the original is because I never forked it when creating this version. I copied file names and that was about it (since at the time I didn't have any modding experience and thought the file names had to be exact for things to work (Which they don't as I've found from future experience and practice). 

I'll provide the source code when possible but sadly I'm not sure when I'll be able to do so due to me being away.
